server/api/routes.js

```diff
@@ -1,10 +1,12 @@
 const express = require('express')
 const router = express.Router()
-const server_db_utils = require('./../server_db_utils')
+const server_db_utils = require('../server_db_utils')
+const sim_db_utils = require('../../simulator/sim_db_utils')
 
 //CONSUMPTION
-//Update prosumer consumption by id
-router.put('/prosumer/:id/consumption', async (req, res) => {
+//Update prosumer.js consumption by id
+router.put('/prosumer.js/:id/consumption', async (req, res) => {
```

Comment:
`prosumer.js`... I guess it's a typo :smile:

Reply:
haha wops, im on it!